crates/rome_js_analyze/src/analyzers/nursery/use_literal_keys.rs

```diff
@@ -0,0 +1,130 @@
+use crate::JsRuleAction;
+use rome_analyze::{context::RuleContext, declare_rule, ActionCategory, Ast, Rule, RuleDiagnostic};
+use rome_console::markup;
+use rome_diagnostics::Applicability;
+use rome_js_factory::make::{ident, js_name, js_static_member_expression, token};
+use rome_js_syntax::{
+    AnyJsExpression, AnyJsLiteralExpression, AnyJsName, JsComputedMemberAssignment,
+    JsComputedMemberExpression, T,
+};
+use rome_rowan::{declare_node_union, AstNode, BatchMutationExt, SyntaxResult, TextRange};
+
+declare_rule! {
+    /// Enforce the usage of a computed expression over a static expression with strings.
```

Comment:
This should not be reversed?

```suggestion
    /// Enforce the usage of a static property access over computed property access.
```